exchange/bidder_test.go

```diff
@@ -73,7 +73,10 @@ func TestSingleBidder(t *testing.T) {
 		bidResponse: nil,
 	}
 
-	ctx := context.Background()
+	var requestTmaxMS int64 = 700
+	deadline := time.Now().Add(time.Duration(requestTmaxMS) * time.Millisecond)
```

Comment:
Please don't use `time.Now()` in unit tests. Always use a predefined time value.